ports/psoc6/main.c

```diff
@@ -179,9 +161,39 @@ int main(int argc, char **argv) {
 
 // TODO: to be implemented
 void nlr_jump_fail(void *val) {
-    mp_printf(&mp_plat_print, "nlr_jump_fail\n");
+    mplogger_print("nlr_jump_fail\n");
+    mp_obj_print_exception(&mp_plat_print, MP_OBJ_FROM_PTR(val));
 
-    while (1) {
-        ;
+    for (;;) {
+        __BKPT(0);
     }
 }
+
+
+const char psoc6_help_text[] =
```

Comment:
This is also provided for rpi or? 
I like it but maybe as the machine module grows this can get too lengthy?

Reply:
maybe we could put it in a separate source file to avoid clutter in the `main.c`. See here: https://github.com/jaenrig-ifx/micropython/blob/ports/psoc6/ports/esp32/help.c

Reply:
done